docs/community/3.9-announcement.md

```diff
@@ -168,7 +168,7 @@ Both `APIView.exclude_from_schema` and the `exclude_from_schema` argument to the
 
 For `APIView` you should instead set a `schema = None` attribute on the view class.
 
-For function based views the `@schema` decorator can be used to exclude the view from the schema, by using `@schema(None)`.
+For function-based views the `@schema` decorator can be used to exclude the view from the schema, by using `@schema(None)`.
```

Comment:
Not convinced.

Reply:
I think this is correct. 

https://dictionary.cambridge.org/us/dictionary/english/based

Reply:
I think we should stick to the version with a hyphen, as in the Django documentation:
https://docs.djangoproject.com/en/2.1/topics/class-based-views/#class-based-views